### ISSUE TITLE
Error: Unknown module 'utils/ErrorMapper'

### ISSUE THREAD

**Issue by rsnitsch** [screepers/screeps-typescript-starter#152]
**tl;dr Over night, all module imports stopped working, even the default ones like utils/ErrorMapper. Using exact same code and commands.**

Yesterday I set up screeps-typescript and moved my old Javascript code to the new codebase and "translated" some of it to Typescript already (adapted for it).

Everything worked fine: Compiling and _pushing_ to main or sim and _running_ the code in Training _and_ the public server. I used Git from the start. I know exactly what the latest version of the code is that I successfully compiled and uploaded yesterday evening.

Today, I start with the exact same codebase as yesterday (as confirmed by Git - no changes!) and use the exact same commands to build the code, but I get error messages all over the place. I don't get it. Yesterday, literally a few hours ago, the exact same code compiled fine.

In the simulator (training), the following error message appears:
```
Error: Unknown module 'utils/ErrorMapper'
    at Object.<anonymous>:2:206024
    at main:5:19
    at eval:64:4
    at Object.<anonymous>:2:206380
    at Object.r.run:2:206889
```

On the commandline, I see that npm run build produces the following warnings/errors:

```
PS C:\dev\screeps> npm run push-sim

> screeps-typescript-starter@3.0.0 push-sim C:\dev\screeps
> rollup -c --environment DEST:sim

cleared:  C:\dev\screeps\dist

src/main.ts → dist/main.js...
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
utils/ErrorMapper (imported by src\main.ts)
roleHarvester (imported by src\main.ts)
role.builder (imported by src\main.ts)
role.guardRanged (imported by src\main.ts)
spawnCreeps (imported by src\main.ts)
role.tower (imported by src\main.ts)
role.upgrader (imported by src\main.ts)
created dist/main.js in 1s
```

I can fix these errors by prepending "./src" to all import statements, e.g. instead of

`import { ErrorMapper } from "utils/ErrorMapper";`

I can use

`import { ErrorMapper } from "./src/utils/ErrorMapper";`

There are two problems with this. First, it was not necessary yesterday. Second, it confuses the Typescript interpreter/linter in my IDE (Visual Studio Code). VS Code complains that the module ./src/utils/ErrorMapper cannot be found. Even if I change the baseUrl in tsconfig.json, it still does not find them.

I tried many things, but I could not fix it. I am grateful for any hints.

Disclaimer: I am an experienced programmer, but I never used Typescript or Rollup or Visual Studio Code for javascript/typescript development.

**Comment by rsnitsch** [comment]
I reduced my src/main.ts to this:

```
import { ErrorMapper } from "utils/ErrorMapper";

export const loop = ErrorMapper.wrapLoop(() => {
  console.log("doing nothing");
});
```

But it still does not work. push-sim output:

```
> rollup -c --environment DEST:sim

cleared:  C:\dev\screeps\dist

src/main.ts → dist/main.js...
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
utils/ErrorMapper (imported by src\main.ts)
created dist/main.js in 1.1s
```

Output in screeps training:

```
Error: Unknown module 'utils/ErrorMapper'
    at Object.<anonymous>:2:206024
    at main:5:19
    at eval:72:4
    at Object.<anonymous>:2:206380
    at Object.r.run:2:206889
```

**Comment by rsnitsch** [comment]
I found the problem... this is beyond stupid. When I open the PowerShell console in another directory and use cd to navigate to the screeps-typescript directory, then rollup will fail on module imports. WTF. If I open the console directly in the correct directory, it works. Again: WTF. Note: It only happens when using PowerShell. cmd.exe is not affected by this issue.

**Comment by djD-REK** [comment]
Wow that's bizarre! You'd think PowerShell would work better. I use
https://cmder.net/ CMDer exclusively on Windows (including inside VS Code),
and I think it's fantastic. Thanks for sharing!

There are tons of little issues out there for Windows developers since
"real developers" switched to Mac 10 years ago 🙄

**Comment by rsnitsch** [comment]
@djD-REK Happy new year!

Yes, this is very bizarre. Another user on Slack tested it on his system. He cannot reproduce my issue. Therefore, it might only be a configuration issue of my Windows installation. Could you test it on your system, too? If you cant reproduce it either, then I'll close this issue.

**Comment by djD-REK** [comment]
Happy New Year to you!

Sorry, but I'm not going to be able to do any coding for the next two
weeks, so I'd only be holding you up. Cheers!

**Comment by rsnitsch** [comment]
I have tested it on another Windows 10 system of mine. I cannot reproduce the issue there. It is clearly a configuration issue of my main Windows machine.

I tried to find the cause with ProcessMonitor (Sysinternals suite), but so far I have found nothing that is suspicious (to the contrary, the node executable is accessing the correct build directory).

**Comment by joshleblanc** [comment]
Ran into this on my machine as well. Telling resolve to use src as the root directory fixed the issue. 
 
 I made a PR, if someone can verify everything still works on their machine. Apologies for the double commit there, not sure what happened
